_includes/neurips/2020/about.neurips.html

```diff
@@ -42,16 +41,16 @@ <h2>Call for Participation</h2>
       <div class="contents">
         <h3>Important Dates</h3>
         <ul>
-            <li><strong>September 15th,  2020:</strong> <a target="_blank" href="https://neurips.cc/Register2"> Registration</a> to conference opens</li>
```

Comment:
Please do not remove the target="_blank" for external links - this allows the link to open in a new tab.